### ISSUE TITLE
Can not install this project from pip or easy_install

### ISSUE THREAD

**Issue by tkliuxing** [arkanister/cmsplugin-bootstrap#2]
I can searched this project in pypi: https://pypi.python.org/pypi/cmsplugin-bootstrap/0.1 , but cannot install this project from pip or easy_install.

```
$ easy_install cmsplugin-bootstrap
Searching for cmsplugin-bootstrap
Reading https://pypi.python.org/simple/cmsplugin-bootstrap/
No local packages or download links found for cmsplugin-bootstrap
error: Could not find suitable distribution for Requirement.parse('cmsplugin-bootstrap')
$ easy_install "cmsplugin-bootstrap"
Searching for cmsplugin-bootstrap
Reading https://pypi.python.org/simple/cmsplugin-bootstrap/
No local packages or download links found for cmsplugin-bootstrap
error: Could not find suitable distribution for Requirement.parse('cmsplugin-bootstrap')
$ pip install "cmsplugin-bootstrap"
Downloading/unpacking cmsplugin-bootstrap
  Could not find any downloads that satisfy the requirement cmsplugin-bootstrap
Cleaning up...
No distributions at all found for cmsplugin-bootstrap
Storing debug log for failure in /Users/ron/.pip/pip.log
```

**Comment by arkanister** [comment]
Hello tkliuxing,

Sorry for the delay, I will correct the record on pypi to enable download support by pip and easyinstall. For now you can download this package using the command `pip install git+https://github.com/arkanister/cmsplugin-bootstrap/archive/master.zip`.

Thanks for the feedback.

**Comment by arkanister** [comment]
Hi tkliuxing,

Support for installation by `pip` and `easy_install` was added.
To download it by pip: `pip install cmsplugin-bootstrap`.
To download it by easy_install: `easy_install cmsplugin-bootstrap`.

The problem that occurred was that I had just recorded the package in pypi and had not done uploading. The correction was only upload the packages.

Again, thanks for the feedback.
Any questions, corrections or sujestão that contribute to the evolution of the package, I put myself entirely to disposal.

Regards ...
